### PR TITLE
Generate OsmAnd Docs Map into docmap.json Issue #2341

### PR DESCRIPTION
**Pull request by RZR-UA**

### CODE REVIEW

**Comment by RZR-UA** [comment]
https://github.com/osmandapp/OsmAnd-Issues/issues/2341

File: main/scripts/docs-map-generator.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+const fs = require("fs-extra");
+
+module.exports = OsmAndDocMapGenerator;
+
+const PATH = "./docs/";
+const URL = "https://osmand.net/docs/";
```
Comment (vshcherb) [inline main/scripts/docs-map-generator.js:6]: Chicken & egg problem, we need a running website to get a structure...

Comment (vshcherb) [reply main/scripts/docs-map-generator.js:6]: CC @alex-osm  - I'm not sure this is the right approach.

**Review by alisa911 — APPROVED** [review]

File: main/scripts/docs-map-generator.js (hunk 1 above)
Comment (alex-osm) [reply main/scripts/docs-map-generator.js:6]: Agree, why do we need osmand.net hardcoded here?

**Pull request merged**